### DISCUSSION TITLE
ServiceNow Developer Opportunity - SECRET clearance

### DISCUSSION THREAD

**Discussion by itrusthcm** [community/community#142879] [Actions]
### Select Topic Area

General

### Body

Hi, Please see 3 ServiceNow Developer opportunities in MD, UT, IL Air Force Base: 

https://itrusthcm.applytojob.com/apply/3AIyy1QIIo/Service-Now-Developer

https://itrusthcm.applytojob.com/apply/6XVw0utFxj/Service-Now-Developer

https://itrusthcm.applytojob.com/apply/2WOQHNT9QR/Service-Now-Developer

**Comment by peachmontt** [comment]
thx

**Comment by steferd6** [comment]
A ServiceNow Developer opportunity requiring SECRET clearance typically involves working with sensitive government or defense projects, ensuring the secure development, configuration, and maintenance of the ServiceNow platform. If you're applying from abroad or returning to the Philippines for government-related opportunities, securing your [NBI Clearance Philippines](https://nbiclearancepro.com/) is a crucial step, as it verifies your criminal record status and supports background checks essential for security clearances.
